distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -221,6 +222,53 @@ Complete the following procedure now that several queries have been executed.
 Based on the browser's configuration, the .xls file will be downloaded or automatically displayed in Excel.
 ====
 
+
+
+
+=== Configuration Export/Import
+==== Saving Configurations
+
+. Launch DDF
+
+. At the Web Admin Console, select any Active Application that you want to save
+
+. Select any Active Application
+
+. Modify the Active Application's configuration
+
+. Click *Save changes* on the configuration that you want to save
+
+. Exit Active Application
+
+==== Export Configuration Settings
+
+. Using the DDF Command Line Console, type in `platform:config-export`. This command creates the exported configuration files that are saved to *<INSTALL_DIRECTORY>/etc/exported*
+
+. Zip up the exported files in <INSTALL_DIRECTORY>/etc/exported
+----
+cd  <INSTALL_DIRECTORY>/etc/exported
+zip exportedFiles.zip *
+----
+
+==== Import Configuration Settings
+
+. Unzip the exported files from a previous installation in the *<INSTALL_DIRECTORY>/etc* directory: 
+`unzip exportedFiles.zip <INSTALL_DIRECTORY>/etc`
+
+. Launch the newly installed DDF. 
+
+. Step through the DDF installation process. The newly installed DDF should have the previous DDF's settings imported.
+
+[IMPORTANT]
+====
```

Comment:
Can you reword this so we don't have "we", "an administrator", etc. I just copied this verbatim from some other place and it needs a little TLC to fit with the rest of the document. You should be able to find other callouts and use those as examples.